src/executables/csql.c

```diff
@@ -2682,11 +2682,19 @@ csql (const char *argv0, CSQL_ARGUMENT * csql_arg)
 	{
 	  client_type = DB_CLIENT_TYPE_ADMIN_CSQL_WOS;
 	}
```

Comment:
if user issue command like
`csql -u dba demodb --write-on-standby --sysadm --skip-vacuum -S`
**--skip-vacuum will** be ignored. Is it intended?

Reply:
I think that it should be occurred error when user issue this command with -S and --write-on-standby (WOS) together, because command with WOS only run on CS mode of standby node.
I will fix it

Reply:
I know you fix arrangement of acceptable option combination, I will review the other side of codes.